File: app/src/main/java/co/tpcreative/supersafe/common/response/SyncCategoriesResponse.java
package co.tpcreative.supersafe.common.response;

import java.io.Serializable;
import java.util.List;

import co.tpcreative.supersafe.common.api.response.BaseResponse;
import co.tpcreative.supersafe.common.entities.MainCategoryEntity;
import co.tpcreative.supersafe.model.MainCategoryModel;

public class SyncCategoriesResponse extends BaseResponse implements Serializable {
    public List<MainCategoryModel>files;
    public MainCategoryModel category;
}

File: app/src/main/java/co/tpcreative/supersafe/common/response/SyncCategoriesResponse.kt
package co.tpcreative.supersafe.common.response
import co.tpcreative.supersafe.common.api.response.BaseResponse
import co.tpcreative.supersafe.model.MainCategoryModel
import java.io.Serializable

class SyncCategoriesResponse : BaseResponse(), Serializable {
    var files: MutableList<MainCategoryModel?>? = null
    var category: MainCategoryModel? = null
}